### DISCUSSION TITLE
返回每个字符的位置信息

### DISCUSSION THREAD

**Discussion by Wxinxiny** [PaddlePaddle/PaddleOCR#12702] [Q&A]
如何返回文本框中每个字符的位置信息text box？

**Comment by wulizhong** [comment]
我也想知道怎么返回每个字符的位置信息

**Comment by SWHL** [comment]
这个目前没有现成的模型使用。如果应用场景比较固定的话，例如都是文档类的。可以在拿到文本行坐标之后，根据单字宽度，推算单字符坐标信息。

**Comment by GreatV** [comment]
可以看一下这个PR
- https://github.com/PaddlePaddle/PaddleOCR/pull/10515
